roles/manage-gcp-infra/tasks/configure-registry-object-storage.yaml

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
Overall, can we scale the registry to 0 before performing all these commands to avoid a lot of trigger actions while running the commands?

Reply:
done